data/json/monsters.json

```diff
@@ -1773,7 +1773,11 @@
         ["GRAB", 7]
     ],
     "description":"Still wearing the tattered remnants of improvised armor and weaponry, it is plain to see that this zombie was once a survivor like you.",
-    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
+    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"],
+    "upgrades":{
+        "into": "mon_zombie_survivor_elite",
```

Comment:
You mean in the same way a zombie grenadier elite creates c4 hacks? 
Sure I can remove it so it won't evolve into the elite. But then it wont ever spawn afaik.